docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1489,7 +1483,7 @@ For more information about the Grafana alerts, refer to [About Grafana Alerting]
 
 ### enabled
 
-Enable or disable Grafana Alerting. If disabled, all your legacy alerting data will be available again, but the data you created using Grafana Alerting will be deleted. Set force_migration=true to avoid deletion of data. The default value is `true`.
+Enable or disable Grafana Alerting. If disabled, all your legacy alerting data will be available again. The default value is `true`.
```

Comment:
```suggestion
Enable or disable Grafana Alerting. If disabled, all legacy alerting data remains available. The default value is `true`.
```

Reply:
This one I think we should leave as-is, since it would alter the meaning. Before disabling, any existing legacy alerting data is not available, so the act of disabling will make it available again.

Reply:
@JacobsonMT Ok, sounds good to me, thanks for taking a look and letting me know.